crates/turborepo-lib/src/cli.rs

```diff
@@ -422,9 +422,11 @@ pub fn run(repo_state: Option<RepoState>) -> Result<Payload> {
         clap_args.cwd = Some(repo_state.root);
     }
 
-    if let Some(cwd) = &clap_args.cwd {
-        clap_args.cwd = Some(fs_canonicalize(cwd)?);
-    }
+    let repo_root = if let Some(cwd) = &clap_args.cwd {
+        fs_canonicalize(cwd)?
```

Comment:
Don't we still need to normalize `clap_args.cwd` so Go gets a normalized path?

Reply:
Ah good catch